Tests/PostgresNIOTests/New/Messages/PasswordTests.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 01.02.21.
+//
+
+import XCTest
```

Comment:
Ditch Xcode template headers.
```suggestion
import XCTest
```